solutions/swb-reference/integration-tests/support/resources/environments/environment.ts

```diff
@@ -13,6 +14,40 @@ export default class Environment extends Resource {
     super(clientSession, 'environment', id, parentApi);
   }
 
+  public async pollEnvironment(
+    pollingIntervalInSeconds: number,
+    maxWaitTimeInSeconds: number,
```

Comment:
Both `pollingIntervalInSeconds` and `maxWaitTimeInSeconds` should come directly from constants instead of method params.

Reply:
> Good work. I'm glad to see that the tests passed. Out of curiosity, how long did it take for this test to run till completion?

a little more than 15 minutes only for environments multiStep, mostly for the start step which takes around 5 minutes and we do it twice.

Reply:
I will add a default polling interval and max waiting time  and make the param optional so we can decide in code if we want to specify or not the polling interval and max waiting time